app/api/v1/access.py

```
@@ -92,14 +92,14 @@ async def add_vault_member(
-        logger.warning("resource_not_found", error=str(e))
+        logger.warning("resource_not_found")
         raise HTTPException(status_code=status.HTTP_404_NOT_FOUND, detail="Resource not found")
     except (UnauthorizedVaultAccessError, InsufficientPermissionsError) as e:
-        logger.warning("access_denied", error=str(e))
+        logger.warning("access_denied")
         raise HTTPException(status_code=status.HTTP_403_FORBIDDEN, detail="Forbidden")
     except ValueError as e:
         # e.g., attempting to add the owner as a member
-        logger.warning("validation_error", error=str(e))
+        logger.warning("validation_error")
         raise HTTPException(status_code=status.HTTP_422_UNPROCESSABLE_CONTENT, detail=str(e))
```

Ruff (0.15.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)